src/phoenix/server/api/input_types/SpanSort.py

```diff
@@ -1,33 +1,62 @@
 from enum import Enum
+from functools import partial
+from operator import attrgetter
+from typing import Any, Iterable, Iterator
 
+import pandas as pd
 import strawberry
-from pandas import DataFrame
 
 from phoenix.core.traces import (
     CUMULATIVE_LLM_TOKEN_COUNT_COMPLETION,
     CUMULATIVE_LLM_TOKEN_COUNT_PROMPT,
     CUMULATIVE_LLM_TOKEN_COUNT_TOTAL,
-    END_TIME,
     LATENCY_MS,
+)
+from phoenix.server.api.types.SortDir import SortDir
+from phoenix.trace.schemas import Span
+from phoenix.trace.semantic_conventions import (
     LLM_TOKEN_COUNT_COMPLETION,
     LLM_TOKEN_COUNT_PROMPT,
     LLM_TOKEN_COUNT_TOTAL,
-    START_TIME,
 )
-from phoenix.server.api.types.SortDir import SortDir
+
+
+def _get_attribute_value(span: Span, key: str) -> Any:
+    return span.attributes.get(key)
 
 
 @strawberry.enum
 class SpanColumn(Enum):
-    startTime = START_TIME
-    endTime = END_TIME
-    latencyMs = LATENCY_MS
-    tokenCountTotal = LLM_TOKEN_COUNT_TOTAL
-    tokenCountPrompt = LLM_TOKEN_COUNT_PROMPT
-    tokenCountCompletion = LLM_TOKEN_COUNT_COMPLETION
-    cumulativeTokenCountTotal = CUMULATIVE_LLM_TOKEN_COUNT_TOTAL
-    cumulativeTokenCountPrompt = CUMULATIVE_LLM_TOKEN_COUNT_PROMPT
-    cumulativeTokenCountCompletion = CUMULATIVE_LLM_TOKEN_COUNT_COMPLETION
+    startTime = attrgetter("start_time")
+    endTime = attrgetter("end_time")
+    latencyMs = partial(
+        _get_attribute_value,
+        key=LATENCY_MS,
+    )
+    tokenCountTotal = partial(
+        _get_attribute_value,
+        key=LLM_TOKEN_COUNT_TOTAL,
+    )
+    tokenCountPrompt = partial(
+        _get_attribute_value,
+        key=LLM_TOKEN_COUNT_PROMPT,
+    )
+    tokenCountCompletion = partial(
+        _get_attribute_value,
+        key=LLM_TOKEN_COUNT_COMPLETION,
+    )
+    cumulativeTokenCountTotal = partial(
+        _get_attribute_value,
+        key=CUMULATIVE_LLM_TOKEN_COUNT_TOTAL,
+    )
+    cumulativeTokenCountPrompt = partial(
+        _get_attribute_value,
+        key=CUMULATIVE_LLM_TOKEN_COUNT_PROMPT,
+    )
+    cumulativeTokenCountCompletion = partial(
+        _get_attribute_value,
+        key=CUMULATIVE_LLM_TOKEN_COUNT_COMPLETION,
+    )
```

Comment:
Using the enum value as the getter lambda feels a bit like a strange DSL that - while makes the code look brief, it feels like a level of brevity and indirection that ultimately is kinda more complex than needed. I'd consider leaving the enum and enum and just have a function that uses the enum as a parameter. That will make the code easier to test, follow, and debug.